src/index.html

```diff
@@ -17,11 +21,401 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.scss"
     />
```

Comment:
The stylesheet link should refer to a CSS file, not an SCSS file. Make sure to compile your SCSS to CSS and link to the resulting CSS file.